blog/content/post/2020-04-03-trenchboot-nlnet-lz-validation.md

```diff
@@ -853,22 +948,29 @@ build.
     1. [Makefile](https://github.com/TrenchBoot/landing-zone/blob/v0.3.0/Makefile#L5L7)
     1. [main.c](https://github.com/TrenchBoot/landing-zone/blob/v0.3.0/main.c#L31#L141)
 
-2. LZ code utilizes SHA256 during measurements.
+2. LZ code utilizes SHA256 algorithm during measurements with TPM2.0.
 
     Repository: [TrenchBoot/landing-zone - tag v0.3.0](https://github.com/TrenchBoot/landing-zone/tree/v0.3.0)
 
     Files:
     1. [sha256.c](https://github.com/TrenchBoot/landing-zone/blob/v0.3.0/sha256.c)
 
-3. LZ implementation of TPM interface cover both TPM2.0 and TPM1.2 and use
+3. LZ code utilizes SHA1 algorithm during measurements with TPM1.2.
+
+    Repository: [TrenchBoot/landing-zone - tag v0.3.0](https://github.com/TrenchBoot/landing-zone/tree/v0.3.0)
```

Comment:
The code from the LZ tag will not work with TPM1.2. It should point to some recent commit?

Reply:
Yes. As soon as @krystian-hebel changes will be merged in upstream, I will adapt branch and commit.

Reply:
@miczyg1 I have changed it to point to exact branch from our fork of LZ. It is in MR to upstream. As soon as it will be merged, I will adapt this information here. Now, I think it is safe to leave it as it is.